enterprise_access/apps/api/filters.py

```diff
@@ -0,0 +1,39 @@
+"""
+Filter backends for Enterprise Access API.
+"""
+
+from django.db.models import Q
+from edx_rbac import utils
+from rest_framework import filters
+
+from enterprise_access.apps.core import constants
+
+
+class IsSubsidyRequesterOrEnterpriseAdmin(filters.BaseFilterBackend):
+    """
+    Filter that returns subsidy requests that were either created for the user or created under
+    enterprises that the user is an admin of.
+    """
+
+    def get_accessible_enterprises_by_role(self, request):
+        return {
+            'admin': utils.contexts_accessible_from_request(request, [constants.REQUESTS_ADMIN_ROLE]),
+            'learner': utils.contexts_accessible_from_request(request, [constants.REQUESTS_LEARNER_ROLE]),
```

Comment:
This dict has a `learner` key but is not used at all later in the function. Where would this be used?

Reply:
It's not, I can get rid of it.